app/static/pages/edit_form.js

```diff
@@ -240,9 +251,17 @@ function SubmitData(submission)
     contentType: 'application/json',
     url:  "/json/" + collection + "/" + formId,
     data: JSON.stringify(submission.data),
-    success: schemaRecordChange
+    success: postSuccess
   })
-   .fail(function(res, statusCode, statusMsg)
+   .fail(postFail);
+  
+  function postSuccess(result)
+  {
+    schemaRecordChange;
```

Comment:
This statement has no effect

Reply:
Removed, and confirmed that everything still working correctly.